### PR TITLE
Explaining more on zonal vs. focal

### PR DESCRIPTION
**Pull request by florisvdh**
This is an attempt to further clarify the differences between zonal and focal operations to the beginner, see #402 . But be critical and do make further corrections!

@jannes-m I saw that you already extended a bit in f803f5f, so you can compare and choose what to do.

### CODE REVIEW

**Comment by jannes-m** [comment]
Thanks for the suggestion and the effort. I have slightly rewritten your draft, let me know if you like it.

File: 04-spatial-operations.Rmd [hunk 1]
```diff
@@ -803,20 +803,24 @@ Chapter \@ref(gis) shows how to access such GIS functionality from within R.
...
+Just like focal operations, *zonal* operations apply an aggregation function to multiple raster cells.
+<!-- Several aspects distinguish zonal from focal operations:
+- A (rectangular) window of neighboring raster cells (e.g., 3x3 cells) usually defines the spatial filter of focal operations.  -->
+However, a second raster, usually a categorical raster, defines the *zonal filters* (or 'zones') in the case of zonal operations as opposed to a predefined neighborhood window in the case of focal operations (see previous Section).
+Consequently, the raster cells defining the zonal filter do not necessarily have to be neighbors.
+<!--Hence zonal filters can take on many more shapes than spatial filters, which always consist of neighbouring cells (in a rectangular or other configuration).-->
 Our grain size raster is a good example (right panel of Figure \@ref(fig:cont-raster)) because the different grain sizes are spread in an irregular fashion throughout the raster.
+Finally, the result of a zonal operation is a summary table grouped by zone which is why this operation is also known as *zonal statistics* in the GIS world.
```
Comment (florisvdh) [inline 04-spatial-operations.Rmd]:
```suggestion
Finally, the result of a zonal operation is a summary table grouped by zone which is why this operation is also known as *zonal statistics* in the GIS world. 
By contrast, focal operations return a raster.
```

Comment (Robinlovelace) [reply 04-spatial-operations.Rmd]: Looks good to me. Also, first time I've seen 'suggestions' in GitHub. That is awesome!

**Comment by florisvdh** [comment]
This seems OK to me @jannes-m, thanks! The main distinction is very clear now.

I would suggest to be a little more explicit about the difference in output between focal and zonal, see my suggestion above. You can rewrite of course.

**Comment by Robinlovelace** [comment]
I'm happy with this suggestion but will let @jannes-m enjoy the 'Commit suggestion' bit, assuming all good.

File: 04-spatial-operations.Rmd (hunk 1 above)
Comment (Robinlovelace) [reply 04-spatial-operations.Rmd]: Hi @florisvdh I do not have permission to commit your suggestion. But please implement it and we can accept this PR I think. Heads-up @jannes-m .

Comment (florisvdh) [reply 04-spatial-operations.Rmd]: Done. Seems like a Github bug to me though, as 'edits from maintainers' is marked for my branch.

**Comment by Robinlovelace** [comment]
Thanks @florisvdh, this looks good to go!

**Comment by jannes-m** [comment]
Thanks!

**Pull request merged**